common/config/azure-pipelines/jobs/fast-ci.yaml

```diff
@@ -35,8 +35,37 @@ pr:
       - "**/CHANGELOG.json"
       - "**/CHANGELOG.md"
 
+resources:
+  repositories:
+    - repository: build-pipeline-scripts
+      type: git
+      name: iModelTechnologies/imodeljs-build-pipeline-scripts
+
 jobs:
+  - job: VeracodeScan
+    condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
+    displayName: Veracode SAST Scan
+    pool:
+      vmImage: ubuntu-latest
+    steps:
+      - template: templates/appsec/veracode/package-veracode.yaml@build-pipeline-scripts
+        parameters:
+          foldersToInclude: "core,editor,domains,example-code,extensions,presentation,test-apps,tools,ui,utils" #note I skipped common, docs, full-stack-tests
```

Comment:
im wondering if we should exclude test-apps

Reply:
I'm not opposed to it. I suppose the goal is to minimize vulnerabilities in our code that we ship out, so it seems reasonable to me to exclude test-apps.